packages/api/shared/Route.ts

```diff
@@ -200,10 +200,10 @@ export default function createRoute<Params>(): RouteBuilder<Params> {
       return this;
     },
     build() {
-      return (req: NextApiRequest, res: NextApiResponse) => {
+      return async (req: NextApiRequest, res: NextApiResponse) => {
         const handler = handlers[req.method ?? ''];
         if (handler) {
-          handler(req, res);
+          await handler(req, res);
```

Comment:
This was the issue with those logs. We have to return a promise from a nextjs handler or next doesn't know that we are still working to send a response